File: LeetCode/283.c
void moveZeroes(int* nums, int numsSize) {
    for(int i=0; i<numsSize-1; i++) {
        if(!nums[i]) {
            int j=i+1;
            while(j<numsSize && !nums[j]) j++;
            if(j<numsSize) {
                nums[i] = nums[j];
                nums[j] = 0;
            }
        }
    }
}

File: LeetCode/283.py
class Solution:
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        for x in range (0, len(nums)-1):  
            if nums[x] == 0:
                for y in range(x+1, len(nums)):
                    if nums[y] != 0:
                        nums[x], nums[y] = nums[y], nums[x] 
                        break
